README.md

```diff
@@ -11,14 +11,32 @@ Bash-it provides a solid framework for using, developing and maintaining shell s
 If you're using the _Bourne Again Shell_ (Bash) on a regular basis and have been looking for an easy way on how to keep all of these nice little scripts and aliases under control, then Bash-it is for you!
 Stop polluting your `~/bin` directory and your `.bashrc` file, fork/clone Bash-it and start hacking away.
 
+- [Contributing](#contributing)
+- [Installation](#installation)
+  - [Install Options](#install-options)
+  - [via Docker](#install-using-docker)
+  - [Updating](#updating)
+- [Help](#help-screens)
+- [Search](#search)
+  - [Syntax](#syntax)
+  - [Searching with Negations](#searching-with-negations)
+  - [Using Search to Enable or Disable Components](#using-search-to-enable-or-disable-components)
+  - [Disabling ASCII Color](#disabling-ascii-color)
+- [Custom scripts, aliases, themes, and functions](#custom-scripts,-aliases,-themes,-and-functions)
+- [Themes](#themes)
+- [Uninstalling](#uninstalling)
+- [Misc](#misc)
```

Comment:
Since the `Misc` heading has quite a few subtopics, wouldn't it be good to show them in the table of contents, too?